### PR TITLE
[TG-174][fix] Fix tab is not active on start page

### PR DESCRIPTION
**Pull request by artem713**

### CODE REVIEW

**Comment by review-ninja** [comment]
<a href="https://app.review.ninja/garage-it/SmartHouse-frontend/pull/60" target="_blank">ReviewNinja

File: src/app/components/header/header.component.js [hunk 1]
```diff
@@ -27,6 +29,11 @@ export class HeaderComponent {
         return this.routes.find(route => route.headerName && route.useAsDefault);
     }
 
+    initialRouteActivated(route) {
```
Comment (Todmy) [inline src/app/components/header/header.component.js]: Probably we able to get rid of this function. Two propositions:
- investigate can we use 'serialize' method in @RouteConfig;
- simply remove '/dashboard' route and refactor navigationRoutes method: 
  `get navigationRoutes() {
      return this.routes.filter(route => route.headerName);
  }`

File: src/app/components/header/header.component.js [hunk 2]
```diff
@@ -4,30 +4,30 @@ import {RouterLink} from 'angular2/router';
...
-import routes from './../../routes';
-
-const selector = 'sh-header';
-
+const HEADER_COMPONENT_SELECTOR = 'sh-header';
+export const NAVIGATION_ROUTES = [
+    { name: 'Dashboard', headerName: 'Dashboard', iconImage: './assets/Dashboard.png' },
+    { name: 'DeviceList', headerName: 'Devices', iconImage: './assets/DeviceList.png' },
+    { name: 'ScenarioList', headerName: 'Scenarios', iconImage: './assets/ScenarioList.png' },
+    { name: 'Help', headerName: 'Help', iconImage: './assets/Help.png' }
```
Comment (kosich) [inline src/app/components/header/header.component.js:12]: ~~`iconImage` need not here~~
upd: ignore that. was wrong

**Comment by kosich** [comment]
⭐

**Pull request merged**